core/include/drivers/zynqmp_csu.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) Foundries Ltd. 2021
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#ifndef __DRIVERS_ZYNQMP_CSU_H_
+#define __DRIVERS_ZYNQMP_CSU_H_
+
+/* CSU registers */
+#define ZYNQMP_CSU_STATUS_OFFSET		0x00
+#define ZYNQMP_CSU_CTRL_OFFSET			0x04
+#define ZYNQMP_CSU_SSS_CFG_OFFSET		0x08
+#define ZYNQMP_CSU_DMA_RESET_OFFSET		0x0c
+#define ZYNQMP_CSU_MULTI_BOOT_OFFSET		0x10
+#define ZYNQMP_CSU_TAMPER_TRIG_OFFSET		0x14
+#define ZYNQMP_CSU_FT_STATUS_OFFSET		0x18
+#define ZYNQMP_CSU_ISR_OFFSET			0x20
+
+#define ZYNQMP_CSU_ISR_PUF_ACC_ERROR_MASK	BIT(12)
```

Comment:
If you work with Xilinx could please ask them to add this to public register map :)

Value itself is used in their github code base -> so matches what it should be.